conans/client/loader.py

```diff
@@ -175,20 +175,21 @@ def _initialize_conanfile(conanfile, profile):
         # Mixing the global settings with the specified for that name if exist
         tmp_settings = profile.processed_settings.copy()
         package_settings_values = profile.package_settings_values
+        ref_str = "%s/%s@%s/%s" % (conanfile.name, conanfile.version,
```

Comment:
Keep the FIXME?

Reply:
Yes, actually I will try to add some tests and check this FIXME in a separate PR

Reply:
https://github.com/conan-io/conan/pull/8281